selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/workspaces/Workspaces.java

```diff
@@ -113,6 +115,18 @@ public Workspaces(
   @FindBy(xpath = Locators.SEARCH_WORKSPACE_FIELD)
   WebElement searchWorkspaceField;
 
+  public void waitErrorNotificationContainsText(String expectedText) {
+    seleniumWebDriverHelper.waitSuccessCondition(
+        driver -> isErrorNotificationContainsText(expectedText));
+  }
+
+  public boolean isErrorNotificationContainsText(String expectedText) {
+    final String textAttribute = "che-error-text";
+    return seleniumWebDriverHelper
```

Comment:
There is ready-to-use method `SeleniumWebDriverHelper#waitAttributeContainsValue(elementLocator, attributeName, expectedValue)`.